test/data/build.update.yaml

```diff
@@ -1,4 +1,5 @@
 # Build Update Example
 status: ABORTED
+statusMessage: STOP
```

Comment:
Can you add a better example message? I'm still a little confused what it would look like.